vowpalwabbit/options.h

```diff
@@ -88,25 +91,36 @@ struct typed_option : base_option
   typed_option& value(T value)

```

Comment:
is this the convention? or should value be called set_value?

Reply:
Looking back set and get would have been better. I'd like to refactor it, but in a separate change

Reply:
yea, i just wanted to see if we're missing somewhere else where we call into value(...) but we didn't use to update the location

Reply:
oh that's a good point